compspec.json

```diff
@@ -22,44 +22,170 @@
       ]
     },
     "input": {
-      "data": {
-        "label": "Features",
+      "covariates":   
+      {
+        "label": "Covariates",
+        "description": "Files containing features other than brain region measurements (e.g., isControl, sex, age) and the binary label to predict.",    
```

Comment:
Description should say:

"Demographic or medical features (e.g., isControl, sex, age)"